src/Microsoft.Graph/Tasks/PageIterator.cs

```diff
@@ -61,6 +62,21 @@ public static PageIterator<TEntity> CreatePageIterator(IBaseClient client, IColl
             };
         }
 
+        /// <summary>
```

Comment:
I don't think you need a full overload if you make your parameter optional, just add the optional parameter to the existing method.